### ISSUE TITLE
Moment Timezone has no data for...

### ISSUE THREAD

**Issue by chocochik92** [moment/moment-timezone#193]
i've just started using moment-timezone 0.3.1 and did npm install and bower install. (i'm also using moment 2.9.0)

I'm trying to convert a date to a different timezone using `moment.tz(date, timezone).format();` 
This error comes up   `Moment Timezone has no data for America/New_York. 
See http://momentjs.com/timezone/docs/#/data-loading/.`

I've gone to that page and I get how you can add timezones. But I already have that timezone info in the data folder with both packed and unpacked json files. So why does this message show up? Do I have to manually add what timezones I want to use?

**Comment by mj1856** [comment]
Works for me.  Here's a screenshot on a fresh empty directory.

![image](https://cloud.githubusercontent.com/assets/1396388/7174050/4d72b7a8-e3b6-11e4-9d87-821200121a99.png)

Yes, I'm on Windows, but that shouldn't matter.  There are no OS-specific dependencies at runtime.

**Comment by mj1856** [comment]
Closing for now.  If you can demonstrate it failing, please do and I'll reopen.  Thanks.

**Comment by pixelchutes** [comment]
FWIW, I recently went through an upgrade from moment 2.4.0 > 2.10.2 (where timezone + tzdata are now combined.) I was able to reproduce this error, but only because I was loading `moment-timezone`, then loading `moment-timezone-with-data` immediately after. _This is obviously not the correct approach._ 

Removing the first instance, leaving only the new combined version resolved this for me. Just mentioning it here incase someone else runs into this issue, make sure you're not unnecessarily redeclaring moment-timezone.
